### PR TITLE
PR: Copy history entries to Editor Issue #3418

### LINKED ISSUE #3418: feature request: copy to editor
Otpen one tries code out in the console.

Nice woule be to have a "copy to Editor" funktion taking last n entried at history into the current cursor position of the Editor.

### PR DESCRIPTION
**Pull request by Formulos**
### Pull Request Checklist

* [X] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
* [X] Based your PR on the latest version of the correct branch (master or 3.x)
* [X] Followed [PEP8](https://www.python.org/dev/peps/pep-0008/) for code style
* [x] Ensured your pull request hasn't eliminated unrelated blank lines/spaces,
      modified the ``spyder/defaults`` directory, or added new icons/assets
* [X] Wrote at least one-line docstrings for any new functions
* [ ] Added at least one unit test covering the changes, if at all possible
* [X] Described your changes and the motivation for them below
* [X] Noted what issue(s) this pull request resolves, creating one if needed
* [X] Included a screenshot, if this PR makes any visible changes to the UI

## Description of Changes
Right clicking on the ipython console now show the option to copy the history log to the editor
it should be noted that the history log is persistent, so you might copy data from previous session. The code copies everything typed in the ipython console up to the last clear command.

image

### Issue(s) Resolved

Fixes #3418

### CODE REVIEW

**Comment by Formulos** [comment]
Sorry about those, they should be fixed now.

**Comment by Formulos** [comment]
What exactly do you have in mind for this issue? Because just selecting n entries from the history log and passing them to the editor seems like coping and pasting them.

File: spyder/plugins/ipythonconsole/widgets/client.py [hunk 1]
```diff
@@ -555,6 +559,20 @@ def clear_console(self):
         """Clear the whole console"""
         self.shellwidget.clear_console()
 
+    @Slot()
+    def copy_console(self):
+        """Copy the inputs from console to editor"""
```
Comment (CAM-Gerlach) [inline spyder/plugins/ipythonconsole/widgets/client.py]: Per PEP 257, docstrings should be a complete sentence end with a period.

**Comment by CAM-Gerlach** [comment]
> Because just selecting n entries from the history log and passing them to the editor seems like coping and pasting them.

I'm not sure how this proposal doesn't fall under the same category, if we set n = entries_in_history; personally, I don't see either as really being that terribly useful since as you mention, either way you can just select, copy and paste them using the keyboard (with Ctrl-A [or Shift/Control-Arrow Keys] Ctrl-C Alt-1 [or your Editor shortcut here] Ctrl-V) about fast as right clicking, navigating to the option and clicking it.

**Comment by CAM-Gerlach** [comment]
You also checked the box reading

```
[x] Added at least one unit test covering the changes, if at all possible
```

but no unit test is in fact provided.

**Pull request closed without merging**